psq/context_local_pubsub_connection_test.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+
+from mock import Mock
+
+from psq.context_local_pubsub_connection import ContextLocalPubsubConnection
+
+
```

Comment:
Yeah, testing multi-threaded code is a headache. I don't think the test I suggested is too big a deal to write , but I also agree its' testing werkzeug more than your own code . But sometimes testing dependencies is worth it, ties into our gcloud testing debate ha.